packages/block-library/src/tag-cloud/edit.js

```diff
@@ -118,10 +118,25 @@ function TagCloudEdit( { attributes, setAttributes } ) {
 
 	const inspectorControls = (
 		<InspectorControls>
-			<PanelBody title={ __( 'Settings' ) }>
-				<VStack
-					spacing={ 4 }
-					className="wp-block-tag-cloud__inspector-settings"
```

Comment:
If we remove this we should also remove the associated CSS which now is no longer used: https://github.com/WordPress/gutenberg/blob/cbe7ef4d597cc28f3ab0fc9c7593b9af8085aa36/packages/block-library/src/tag-cloud/editor.scss#L13-L19